portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,4 +65,30 @@ private String convertToJsonUsingGson(ArrayList<String> listOfStrings) {
     String json = gson.toJson(listOfStrings);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the comment input from the form.
+    String comment = getParameter(request, "text-input", "");
```

Comment:
Consider checking for an empty string here so you don't store any empty strings

Reply:
Will do, thanks for the feedback.